app/src/main/java/com/codesoom/assignment/service/TaskService.java

```diff
@@ -0,0 +1,58 @@
+package com.codesoom.assignment.service;
+
+import com.codesoom.assignment.entity.Task;
+import com.codesoom.assignment.repository.TaskRepository;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Objects;
+
+@Service
+public class TaskService {
+    private Long lastId = 0L;
```

Comment:
```suggestion
    private Long lastId = 0L;
```
현재 저장되는 id값이 service에 있습니다. 제 생각으로는 repository에서 관리를 해야 할 것 같은데   
repository의 save메소드에서 마지막으로 저장되야할 id값을 1증가시킨후 저장하는 방식으로 하는게 더 맞는 방식일까요?
repository의 save메서드에서 lastId를 증가시키는 일까지 하면 2가지일을 하는 느낌이 드는데 저장자체가 다음 id값으로 저장해야되는게 맞으니 save로 들어가도 될지 궁금합니다.

Reply:
id를 1 증가시키는 동작이 어느 쪽의 책임인지에 대한 문제입니다. id 발행만을 책임지는 서비스를 추가하는 방법도 생각해 볼 수 있고, 말씀하신 것처럼 repository에서 lastId 를 관리하는 방법도 고려해볼 수 있습니다. `save` 메소드에 `id` 증가 로직을 격리하면 `id` 증가의 로직을 `TaskService`에서 제거하게 되어, `TaskService`는 `Task`를 관리하는 일에만 집중할 수 있겠죠.